.github/workflows/ci.yml

```
@@ -264,7 +264,7 @@ jobs:
       - name: Run Integration Tests
         if: ${{ steps.task-status.outputs.affected == '1' }}
         id: integration-tests
-        run: sudo npm install -g pnpm && sudo --preserve-env pnpm turbo test:integration:${{ matrix.test-name }} $TURBO_ARGS
+        run: pnpm turbo test:integration:${{ matrix.test-name }} $TURBO_ARGS
```

What if we sudo without installing `pnpm` explicitly

```suggestion
        run: sudo --preserve-env pnpm turbo test:integration:${{ matrix.test-name }} $TURBO_ARGS
```

---

Then `pnpm` won't be installed. I tried this originally when migrating to `pnpm`